web/login.html

```diff
@@ -58,9 +58,9 @@
 
 		// load main module
 		app.require('main', function () {
-			$('#login, #password').on('keypress', function () {
```

Comment:
I do see that it is missing here event argument for the function. I will add it.